_literate/A-model-tuning.jl

```diff
@@ -27,11 +27,18 @@ r   = range(dtc, :max_depth, lower=1, upper=5)
 # For hyperparameters of type `<:Real`, you should specify a range of values as done above.
 # For hyperparameters of other type (e.g. `Symbol`), you should use the `values=...` keyword.
 #
-# Once a range of values has been defined, you can then wrap the model in a `TunedModel` specifying the tuning strategy. 
-# A tuning strategy consists of (i) a model, (ii) a set of values for the tuned paramters and (iii) a measure to optimize:
+# Once a range of values has been defined, you can then wrap the model in a `TunedModel` specifying the tuning strategy.
 
 tm = TunedModel(model=dtc, ranges=[r, ], measure=cross_entropy)
 
+# Note that "wrapping a model in a tuning strategy" as above means creating a new "self-tuning" version of the model, tuned_model = TunedModel(model=...), in which further key-word arguments specify:
+# 1. the algorithm (a.k.a., tuning strategy) for searching the hyper-parameter space of the model (e.g., tuning=Random(rng=123)or tuning=Grid(goal=100)).
```

Comment:
please use backticks around code `` ` ``

Reply:
I corrected it in second commit.